docs/use_cases/retrieval_augmented_generation_eval.md

```diff
@@ -0,0 +1,181 @@
+
+RAG has received significant attention due to its ability to enhance content generation by leveraging existing information effectively. 
+Its capacity to amalgamate specific, relevant details from multiple sources and generate accurate and relevant content has a lot of potential in various domains like content creation, question & answer application, and information synthesis.
+
+Read more: [here](https://hub.superlinked.com/retrieval-augmented-generation)
+
+## Why do we need RAG?
+
+RAG plays a crucial role to significantly enhance [vector search](https://qdrant.tech/documentation/overview/vector-search/) with the power of Large Language Model (LLM), enabling dynamic content generation based on the retrieved knowledge.
+
+It proves invaluable for tasks requiring detailed, coherent explanations, summaries, or responses that transcend the explicit data stored in vectors. 
+While vector search efficiently finds similar items, RAG takes a step further, offering content synthesis and a deeper level of understanding.
+
+If the question is : "When is vector search alone not enough?" 
+RAG becomes crucial when users seek to generate new content rather than interact with documents or search results directly. 
+It excels in providing contextually rich, informative, and human-like responses, making it an ideal solution for a comprehensive and versatile approach when integrated with vector search.
+
+Your next step is to conduct feasibility studies and correlate the results with your business needs and value expectations.
+
+## Evaluating Retrieval-Augmented Generation
+
+Our experience scaling RAG from Proof-of-concept (POC) to production taught us
+about the unknowns. The lesson is to improve and optimize your solution through evaluation.
+
+### Why do you need to evaluate?
+
+Evaluation is key. It distills a sense of Trust in your application,
+which establishes reputation, and boosts team morale and confidence.
+It also validates that your applications avoid common pitfalls.
+
+As we know, LLMs can make mistakes!! 🙂
+
+One problem is that the 'data' frequently changes. Evaluation helps ensure the results are consistent with user expectations.
+
+> If you can't quantify it, you can't improve it!!
+
+For RAG it can be restated as :
+
+>If you can't retrieve it, you can't generate it!!
+
+Our experience has taught us that it is better to evaluate each component in isolation.
+
+As for different components of RAG viz. Information Retrieval - Context Augmentation - Response Generation , we classified challenges of RAG as below :
+
+<figure>
+<img src=/assets/use_cases/retrieval_augmented_generation_eval/rag_challenges.jpg alt="Classification of Challenges of RAG Evaluation" data-size="100" />
+<figcaption align="center">Slide from RAG Evaluation Presentation mentions ['Lost in the Middle' problem](https://arxiv.org/abs/2307.03172)</figcaption>
```

Comment:
I think it's impossible to put a markdown link in an HTML snippet as it's not rendered properly.

```suggestion
<figcaption align="center">Slide from RAG Evaluation Presentation mentions <a href="https://arxiv.org/abs/2307.03172">'Lost in the Middle' problem</a></figcaption>
```